test/libsolidity/syntaxTests/indexing/array_multidim_rational.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
+  function f() public {
+    bytes[32] memory a;
+    a[8**90][8**90][8**90*0.1];
+  }
+}
```

Comment:
This is failing with an assertion, to be fixed by this PR.

Reply:
Done.